shell/pages/c/_cluster/apps/charts/install.vue

```diff
@@ -1546,9 +1542,12 @@ export default {
             {{ t('catalog.install.error.legacy.enableLegacy.goto') }}
           </nuxt-link>
         </template>
+        <template v-else-if="mcapp">
```

Comment:
This kind of wiggles around the content from the first span

![image](https://user-images.githubusercontent.com/18697775/191946354-d78d3293-b2cd-4fa2-9e84-8673f49983ed.png)

Can we gate the first `span` of this on `!mcapp`?